app/core/ingestion/validator.py

```diff
@@ -54,10 +61,90 @@ def _check_geo_in_db(row_num: int, db: Session, geo_iso: str) -> CheckResult:
     return Result()
 
 
-def validate_document_row(
+def validate_unfccc_document_row(
     db: Session,
     context: IngestContext,
-    row: DocumentIngestRow,
+    row: UNFCCCDocumentIngestRow,
+    taxonomy: Taxonomy,
+) -> None:
+    """
+    Validate the constituent elements that represent this law & policy document row.
+
+    :param [IngestContext] context: The ingest context.
+    :param [DocumentIngestRow] row: DocumentIngestRow object from the current CSV row.
+    :param [Taxonomy] taxonomy: the Taxonomy against which metadata should be validated.
+    """
+
+    errors = []
+    n = row.row_number
+
+    # don't validate: md5sum: str
+    # don't validate: collection_name: str
+    # don't validate: collection_id: str
+    # don't validate: family_name: str
+    # don't validate: document_title: str
+    # don't validate: documents: str
+    # don't validate: author: str
+    # don't validate: geography: str
+    # don't validate: date: datetime
```

Comment:
are the columns configurable? I think we're gonna be adding/removing things for the next couple of days

Reply:
Yup - there's a bit of work when we change stuff but minimal